geocoding/lib/geocoding.dart

```diff
@@ -36,13 +35,28 @@ Future<List<Location>> locationFromAddress(
 /// When not supplied the currently active locale of the device will be used.
 /// The `localeIdentifier` should be formatted using the syntax:
 /// [languageCode]_[countryCode] (eg. en_US or nl_NL).
-Future<List<Placemark>> placemarkFromCoordinates(
+Future<List<Placemark>>? placemarkFromCoordinates(
```

Comment:
Don't change the return type to a "nullable" type.

```suggestion
Future<List<Placemark>> placemarkFromCoordinates(
```